src/simulator/cube.py

```diff
@@ -23,28 +23,37 @@ def bounding_box(self, position: Position) -> Box:
         l = self.length
         w = self.width
         h = self.height
+
         # Because the cube has symmetry, we only need the top side points of the disk.
         # We calculate the distance of the farthest points and then get the dimensions
         # of the bounding box. Note, since we used the top points, because of symmetry,
         # we double the dimensions of the bounding box so that the box surrounds the bottom
         # points of the disk.
         points = [[l / 2, w / 2, h / 2], [-l / 2, w / 2, h / 2],
                   [l / 2, -w / 2, h / 2], [-l / 2, -w / 2, h / 2]]
-        new_x = list()
-        new_y = list()
-        new_z = list()
+
+        # like [x-points, y-points, z-points]
```

Comment:
I had something else in mind actually. something like new_points.append(new_point).
But not that important now that I think again. This is fine.